pdf_build_src/process_markdowns.py

```diff
@@ -141,6 +141,164 @@ def modify_changelog():
         file.writelines(data)
 
 
+"""Number of chars maximal in one line approximated from a line of the PDF"""
+NB_CHARS_LINE_PDF = 100
+
+def correct_table(table):
+    """Compute the max number of characters and dashes for each column and create the corrected table
+
+    Parameters
+    ----------
+    table : List of List of str
+        Table content extracted from the markdown file.
+
+    Returns
+    -------
+    new_table : List of List of str
+        List of corrected lines of the input table with corrected number of dashes and aligned fences.
+        To be later join with |'s
+    """
+    import numpy as np
```

Comment:
I think you can import this at the top of the file, with a blank line after the imports of the inbuilt libraries

or what's the need for an import at the function level here?